worldedit-core/src/main/java/com/sk89q/worldedit/util/command/SimpleDispatcher.java

```diff
@@ -177,13 +177,8 @@ public SimpleDescription getDescription() {
 
     @Override
     public boolean testPermission(CommandLocals locals) {
-        for (CommandMapping mapping : getCommands()) {
-            if (mapping.getCallable().testPermission(locals)) {
-                return true;
-            }
-        }
-
-        return false;
+        // checking EVERY perm here doesn't do anything but waste cpu
+        return true;
```

Comment:
What was happening here is that WorldEdit was checking every permission in the class, and if you didn't have any of them it would exit early. 

The indifidual commands check the permission already.